<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>snow_widget</class>
 <widget class="QWidget" name="snow_widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>470</width>
    <height>596</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>snow_widget</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="2" colspan="3">
    <widget class="QComboBox" name="cb_departement">
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Zone de neige :</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Altitude (entre 0 et 2000m) :</string>
     </property>
    </widget>
   </item>
   <item row="5" column="4">
    <widget class="QLineEdit" name="le_charge_caract"/>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>pente de la toiture (degres) :</string>
     </property>
    </widget>
   </item>
   <item row="4" column="3" colspan="2">
    <widget class="QLineEdit" name="le_altitude_2"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Département :</string>
     </property>
    </widget>
   </item>
   <item row="6" column="4">
    <widget class="QLineEdit" name="le_charge_except"/>
   </item>
   <item row="5" column="0" colspan="4">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Charge neige caractéristique (kN/m2) :</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2" colspan="3">
    <widget class="QLineEdit" name="le_zone"/>
   </item>
   <item row="3" column="2" colspan="3">
    <widget class="QLineEdit" name="le_altitude"/>
   </item>
   <item row="6" column="0" colspan="3">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Charge neige exceptionnelle (kN/m2) :</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2" colspan="3">
    <widget class="QComboBox" name="cb_region">
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Région :</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="5" alignment="Qt::AlignHCenter">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label_6">
        <property name="maximumSize">
         <size>
          <width>400</width>
          <height>400</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>images/carte_fr.jpg</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
